node/index.ts

```diff
@@ -50,8 +48,6 @@ export default new Service<Clients, RecorderState, CustomContext>({
       },
       biggySearch: {
         concurrency: 10,
-        memoryCache: biggySearchCache,
-        timeout: SIX_SECONDS_MS,
```

Comment:
why did you remove the `timeout`?

Reply:
Thought the timeout was related to the cache.

Reply:
I think this timeout is for the existing requests on this client, not for the cache, but it would be nice to check.. 🤔

Reply:
According to Athos this is the request timeout. I restored the line.